src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -291,6 +323,20 @@ paths:
               $ref: '#/definitions/ToolClass'
 
 definitions:
+  ToolFile:
+    type: object
+    properties:
+      path: 
+        type: string 
+        description: Relative path of a file that can be used with the GA4GH .../{type}/descriptor/{relative_path} endpoint if the file is a primary or secondary descriptor
```

Comment:
What can be done with the dockerfile and test files? (as a question that comes up when reading this and could be useful to document)